stages/org.osbuild.yum.config

```diff
@@ -26,13 +28,73 @@ SCHEMA = r"""
         "description": "Determines how upstream HTTP caches are instructed to handle any HTTP downloads that YUM does."
       }
     }
+  },
+  "plugins": {
+    "additionalProperties": false,
+    "type": "object",
+    "description": "YUM plugins configuration",
+    "properties": {
+      "langpacks": {
+        "additionalProperties": false,
+        "type": "object",
+        "description": "'langpacks' YUM plugin configuration",
+        "properties": {
+          "locales": {
+            "type": "array",
+            "description": "list of locales for YUM",
+            "items": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
   }
 }
 """
 
 
+def configure_plugins(tree, plugins_options):
+    for plugin, plugin_options in plugins_options.items():
+        # don't touch the configuration file if there is nothing to do
+        if not plugin_options:
+            continue
```

Comment:
Alternatively you could use `minProperties` in the `langpacks` object schema to ensure that at least one property is provided.